src/makeCalculator.js

```diff
@@ -37,7 +37,48 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  let result = 0;
+
+  const calculator = {
+    add: (number) => {
+      result += number;
+
+      return calculator;
+    },
+    subtract: (number) => {
+      result -= number;
+
+      return calculator;
+    },
+    multiply: (number) => {
+      result *= number;
+
+      return calculator;
+    },
+    divide: (number) => {
+      if (result === 0) {
```

Comment:
The condition to check if the result is zero isn't correct. It should check if the input number is zero before performing division operation, not the result variable.